File: pc_receiver/wifi_receiver.js
// Windows 版 Node.js TCP 接收器
// 安装依赖：npm install robotjs
// 运行：node wifi_receiver.js

const net = require('net');
const robot = require('robotjs');

const HOST = '0.0.0.0';
const PORT = 8888;

const server = net.createServer(socket => {
  console.log(`[连接] ${socket.remoteAddress}:${socket.remotePort}`);

  socket.on('data', data => {
    const text = data.toString('utf8').replace(/[\r\n]+$/, '');
    console.log(`[收到] ${text}`);
    // 模拟键盘输入到当前光标位置
    robot.typeString(text);
    // 可选：发送 Tab 键
    // robot.keyTap('tab');
  });

  socket.on('end', () => console.log('[断开]'));
  socket.on('error', err => console.error('[错误]', err.message));
});

server.listen(PORT, HOST, () => {
  console.log(`[监听] ${HOST}:${PORT} - 等待手机连接...`);
  console.log('提示：将手机和电脑连接同一WiFi，在App中填写本机IP地址');
});

File: pc_receiver/wifi_receiver.py
/**
 * WiFi 传输说明（电脑端配置）
 *
 * 手机通过 TCP Socket 向电脑发送数据，电脑需运行一个简单的 TCP 监听服务，
 * 接收到数据后模拟键盘输入（写入当前光标位置）。
 *
 * ─── Python 版（推荐，跨平台）────────────────────────────────────────────────
 *
 * 安装依赖：pip install pyautogui
 *
 * 运行：python wifi_receiver.py
 *
 * ─────────────────────────────────────────────────────────────────────────────
 */

import socket
import pyautogui
import threading

HOST = '0.0.0.0'
PORT = 8888          # 与 App 设置中的端口一致

def handle_client(conn, addr):
    print(f"[连接] {addr}")
    with conn:
        while True:
            data = conn.recv(1024)
            if not data:
                break
            text = data.decode('utf-8').rstrip('\n').rstrip('\r')
            print(f"[收到] {text}")
            pyautogui.typewrite(text, interval=0.02)
            # 如果需要 Enter 键：pyautogui.press('enter')

with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
    s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
    s.bind((HOST, PORT))
    s.listen()
    print(f"[监听] {HOST}:{PORT} - 等待手机连接...")
    while True:
        conn, addr = s.accept()
        threading.Thread(target=handle_client, args=(conn, addr), daemon=True).start()
